src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2791,7 +2795,6 @@ private void EmitConstantExpression(TypeSymbol type, ConstantValue constantValue
             }
         }
 
-        //PROTOTYPE(readonly-ref): check the callers of this method, they may want to call EmitDefaultValue instead for its optimizations.
```

Comment:
Looks like no issue was opened for this PROTOTYPE comment. Can you explain why it is no longer relevant?

Reply:
I checked the callers as the comment asks. SO this is no longer actionable.

---
In reply to: [139487907](https://github.com/dotnet/roslyn/pull/21982#discussion_r139487907) [](ancestors = 139487907)

Reply:
None of the remaining callers would be better with calling EmitDefaultValue since none of the optimizations that EmitDefaultValue does on top of EmitInitObj would be applicable.

---
In reply to: [139513508](https://github.com/dotnet/roslyn/pull/21982#discussion_r139513508) [](ancestors = 139513508,139487907)